acceptance-test/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,9 @@
+spring:
+datasource:
+  driver-class-name: org.h2.Driver
+  url: jdbc:h2:mem:testdb;MODE=PostgreSQL;INIT=CREATE SCHEMA IF NOT EXISTS TEST
```

Comment:
Instead of TEST, I was wondering if we need to rename it to EXAMPLE. What do you think ?

Reply:
Yes we can rename it to domain specific.
However we need to override this With actual domain selected during app generation.

Reply:
I'll check and make the change. Thanks for feedback.